pkg/resourcegroup/controllers/root/root_controller.go

```diff
@@ -163,7 +163,8 @@ func (r *Reconciler) reconcileDisabledResourceGroup(ctx context.Context, req ctr
 	return r.reconcile(ctx, req.NamespacedName, []v1alpha1.ObjMetadata{}, true)
 }
 
-func isStatusDisabled(resgroup *v1alpha1.ResourceGroup) bool {
+// IsStatusDisabled returns whether the ResourceGroup has disabled status updates.
+func IsStatusDisabled(resgroup *v1alpha1.ResourceGroup) bool {
```

Comment:
nit: Is there a better place for this than under the controllers package? Like maybe under pkg/resourcegroup?

Reply:
Done